pwndbg/config.py

```diff
@@ -62,10 +62,13 @@ def __init__(self, name, default, docstring):
                                         gdb.COMMAND_SUPPORT,
                                         getParam(default))
         self.value = default
+        self.trigger = trigger
 
         setattr(module, self.name, self)
 
     def get_set_string(self):
+        if self.trigger:
```

Comment:
tl;dr I like the idea, but we can extend it to a general case (N triggers vs 1 trigger) and also make it easier to register triggers